service/worker/handler/bucket_handlers.go

```diff
@@ -30,9 +32,10 @@ import (
 	"github.com/hibiken/asynq"
 	mclient "github.com/minio/minio-go/v7"
 	"github.com/rs/zerolog"
-	"strings"
 )
 
+var defaultRegion = "us-east-1"
```

Comment:
I don't think it is a good idea to have a region hard coded.
Even if it is used as fallback.

Reply:
done, moved to config